benchmark/snn_llm/qwen2.py

```
@@ -0,0 +1,666 @@
+@dataclass(frozen=True, kw_only=True)
+class Qwen2Config(ModelConfig):
+    builder: ClassVar[str] = "benchmark.snn_llm.qwen2.Qwen2Builder"
+    source_path: Path
+    calibration_path: Path
+
+
+Qwen2Config.__init__.__doc__ = r"""Configure Qwen2 QCFS-SG fine-tuning.
+
+**API Language** - 中文 | English
+
+**中文：** 组合 MCore Transformer、Qwen2 模型事实、Hugging Face 权重目录和
```

Ruff (0.16.1)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)